src/commands/new.ts

```diff
@@ -13,6 +13,13 @@ import {
   clearSpinners,
 } from "../tools/pretty"
 
+// CLI tool versions we support
+const cliDependencyVersions: { [k: string]: string } = {
```

Comment:
Should we put this somewhere other than `new.ts`? Maybe in `/commands`? Seems like these could be used elsewhere in the CLI.

Reply:
Good question. Perhaps in here: https://github.com/infinitered/ignite/tree/master/src/tools

Reply:
I do tend to be partial to "co-locate until you can't" though, tbh.